src/libraries/Microsoft.Extensions.Logging.Abstractions/gen/Microsoft.Extensions.Logging.Generators.csproj

```diff
@@ -9,6 +9,7 @@
     <CLSCompliant>false</CLSCompliant>
     <IsPackable>false</IsPackable>
     <AnalyzerLanguage>cs</AnalyzerLanguage>
+    <AutoGenerateAssemblyVersion>true</AutoGenerateAssemblyVersion>
```

Comment:
I guess we didn’t have a property to identify any analyzer and apply this centrally?  It means analyzers/generators need to all set this property.

Reply:
We should already have one which you introduced Eric. I think it's called IsAnalyzerProject

Reply:
IsGeneratorProject

Reply:
Interesting, I had thought we had it but when I looked for it the other day with @Anipik I didn't find it.  I forget why I concluded that but glad we have this.